process/gazette.py

```diff
@@ -0,0 +1,32 @@
+from typing import List, Optional
+
+from .etl.file_extract import *
+
+
+class Gazette:
+    """
+        The Gazette class contains the methods to process all gazette
+        editions downloaded in project querido diário.
+
+        Args:
+            filepaths: a list of gazette editions to process or read
+            apache_tika_jar: a filepath pointing to the apache tika
+                             installation.
+            metadata: a flag for extracting metadata rather than text
```

Comment:
Is not better have a single object for each edition? I think this class should represent a edition in memory. I think we can aggregate editions in other ways...

Reply:
that's a good point. besides that's the only time we'll really need parallel processing, so we wouldn't add any new functionality here. i'll make and push changes to reflect these.